### PR TITLE
Added NGAME and games invoked by it.

### PR DESCRIPTION
**Pull request by eswenson1**

### CODE REVIEW

File: src/games/lsrrtn.1 [hunk 1]
```diff
@@ -0,0 +1,140 @@
+title LLSRTN - LISP interface to -*-MIDAS-*- inquir lookups
```
Comment (larsbrinkhoff) [inline src/games/lsrrtn.1]: There already is a INQUIR; LSRRTN 31.  Is this one different?

File: src/games/stone.extra [hunk 2]
```diff
@@ -0,0 +1,105 @@
+
```
Comment (larsbrinkhoff) [inline src/games/stone.extra]: STONE EXTRA seems to be a personal resume rather than game documentation.

File: src/games/stone.next [hunk 3]
```diff
@@ -0,0 +1,95 @@
+
```
Comment (larsbrinkhoff) [inline src/games/stone.next]: Another version of Mr Cohen's resume.

**Comment by eswenson1** [comment]
I wonder how I got those two stone.(extra next) in there?  I removed them. The version of LSRRTN is different in INQUIR and GAMES. I kept the one in GAMES local to GAMES for now. I'll enter an issue to compare the two and attempt to consolidate if appropriate.

**Comment by larsbrinkhoff** [comment]
Thanks for fixing my LIMERI mistake!

**Pull request merged**